### PR TITLE
Configuration example for headless API-driven CMS Directus

### PR DESCRIPTION
**Pull request by LiamMartens**

### CODE REVIEW

File: directus/Caddyfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# change to the hostname you want to bind to
+localhost:3000
+
+# change to correct document root
+# in this example config directus is assumed to be root
+# you may need to adjust rewrites according to your own setup
+root /var/www/directus
+
+# Set the path to the php-fpm process.
```
Comment (mholt) [inline directus/Caddyfile]: The path is probably supposed to stay `/` but I imagine you mean "set the host and port to..."

File: directus/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Directus on Caddy
+Caddy configuration for the headless API-driven CMS directus.
```
Comment (mholt) [inline directus/README.md]: Can you make the word "directus" a link?

File: directus/Caddyfile (hunk 1 above)
Comment (LiamMartens) [inline directus/Caddyfile]: I actually mean if you run directus in a subdirectory which is quite common since it's a headless CMS

Comment (mholt) [inline directus/Caddyfile]: Oh, okay. In that case, maybe rephrase this; the path doesn't point to the php-fpm process.

Comment (LiamMartens) [inline directus/Caddyfile]: Oh ye okay well I may have kind of copied that comment, should I rephrase?

Comment (mholt) [inline directus/Caddyfile]: Yeah, I've been more critical lately :smile: Sorry.

A suggestion to rephrase to: "Set the base path that accesses Directus on your site and the address to php-fpm" or something.

**Comment by mholt** [comment]
Thank you!
